governance/xc_admin/packages/crank_pythnet_relayer/src/index.ts

```diff
@@ -177,7 +178,12 @@ async function run() {
             })
             .remainingAccounts(extraAccountMetas)
             .preInstructions(preInstructions)
-            .rpc({ skipPreflight: true });
+            // Use a high compute unit limit to avoid running out of compute units
+            // as some operations can use a lot of compute units.
+            .postInstructions([
+              ComputeBudgetProgram.setComputeUnitLimit({ units: 1000000 }),
+            ])
+            .rpc({ skipPreflight: false });
```

Comment:
I like skip preflight better because it leaves a trace in the blockchain (as a landed failed tx)